beamer/health/check.py

```diff
@@ -389,6 +441,26 @@ def analyze_transfers(transfers: TransferMap, ctx: Context) -> None:
         else "Everything is calm in Beamerland."
     )
 
+    def format_token_volume(token_amount: int, token_details: TokenDetails) -> str:
+        return (
+            str(get_token_amount_in_decimals(token_amount, token_details))
+            + " "
+            + token_details["symbol"]
+        )
```

Comment:
I would move this out to module level. It does not depend on anything around it.